### ISSUE TITLE
Builds sometimes fail with with "no space left on device"

### ISSUE THREAD

**Issue by gpelouze** [QCDIS/NaaVRE-flavors#1]
We get the error message:

```
Error: buildx failed with: ERROR: failed to solve: failed to copy to tar: rpc error: code = Unknown desc = write /tmp/naavre-jupyter-laserfarm.tar: no space left on device
```

This already happened https://github.com/QCDIS/NaaVRE. While it happens less often here, it didn’t entirely disappear. 

If it cannot be solved on GitHub runners, it could require to use self-hosted ones for building `naavre-jupyter-*`.

**Comment by gpelouze** [comment]
We can try:


```yaml
    - name: Free Up GitHub Actions Ubuntu Runner Disk Space
      uses: jlumbroso/free-disk-space@main
      with:
        # This might remove tools that are actually needed, if set to "true" but frees about 6 GB
        tool-cache: false
        # All of these default to true, but feel free to set to "false" if necessary for your workflow
        android: true
        dotnet: true
        haskell: true
        large-packages: true
        swap-storage: true
```

**Comment by gpelouze** [comment]
This also exists: https://github.com/marketplace/actions/maximize-build-disk-space

**Comment by gpelouze** [comment]
We also have the issue on `Push naavre-jupyter-laserfarm`

**Comment by gpelouze** [comment]
Fixed by #4
